accession_register.py

```diff
@@ -54,17 +54,17 @@ def main(args_):
     Launches functions that will generate a helper accessions register
     '''
     args = parse_args(args_)
-    register_dict = ififuncs.extract_metadata(args.register)[0]
-    technical_dict = ififuncs.extract_metadata(args.technical)[0]
-    filmographic_dict = ififuncs.extract_metadata(args.filmographic)[0]
-    for accession in register_dict:
+    sorted_csv_dict = ififuncs.extract_metadata(args.sorted_csv)[0]
+    pbcore_csv_dict = ififuncs.extract_metadata(args.pbcore_csv)[0]
+    filmo_csv_dict = ififuncs.extract_metadata(args.filmo_csv)[0]
+    for accession in sorted_csv_dict:
         number = accession['accession number']
-        for technical_record in technical_dict:
+        for technical_record in sorted_csv_dict:
```

Comment:
This looks unintentional? Changing the dictionary names may be outside of the scope of this, and it appears that this will search for tech records in the sorted accessions register rather than the pbcore CSV.

Reply:
Actually changing the dict names makes sense, but `sorted_csv_dict` should be `pbcore_csv_dict` - i'm guessing?

Reply:
ok, - great, thanks, I can change that - sorted_csv_dict to pbcore_csv_dict

Reply:
Cool, you should be able to edit it directly in the text editor on github in your branch I think. Lemme know if you're having issues with the editing.